helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -171,6 +159,10 @@ generic-service:
       HMPPS_SQS_QUEUES_ALERTS_QUEUE_NAME: "sqs_queue_name"
     domain-events-sqs-nomis-update-alerts-dlq:
       HMPPS_SQS_QUEUES_ALERTS_DLQ_NAME: "sqs_queue_name"
+    domain-events-sqs-nomis-update-casenotes:
+      HMPPS_SQS_QUEUES_CASENOTES_QUEUE_NAME: "sqs_queue_name"
+    domain-events-sqs-nomis-update-casenotes-dlq:
+      HMPPS_SQS_QUEUES_CASENOTES_DLQ_NAME: "sqs_queue_name"
```

Comment:
Does this mean it's going to be turned on in dev? Before the integration tests are written?